### PR TITLE
fix: document HDMI problem on RPi 4

### LINKED ISSUE #3414: Make both HDMI ports work on Raspberry Pi 4
It seems that due to `hdmi_safe=1` in config.txt (see https://github.com/talos-systems/talos/pull/3115), only one HDMI port works, the other shows the rainbow. It should be easy to fix by settings that option for both ports. See https://www.raspberrypi.org/documentation/configuration/config-txt/video.md and https://www.raspberrypi.org/documentation/configuration/config-txt/conditional.md

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #3414.

- [x] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [x] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [x] you generated documentation (`make docs`)
- [x] you ran unit-tests (`make unit-tests`)

### CODE REVIEW

File: website/content/docs/v0.10/Single Board Computers/rock64.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ xz -d metal-rock64-arm64.img.xz
 
 ## Writing the Image
 
-The path to your SD card can be found using `fdisk` on Linux or `diskutil` on Mac OS.
-In this example we will assume `/dev/mmcblk0`.
+The path to your SD card can be found using `fdisk` on Linux or `diskutil` on macOS.
+In this example, we will assume `/dev/mmcblk0`.
```
Comment (smira) [inline website/content/docs/v0.10/Single Board Computers/rock64.md:32]: I believe comma is not required here but not a native English speaker ;)

Comment (AlekSi) [reply website/content/docs/v0.10/Single Board Computers/rock64.md:32]: Grammarly asked me to add it, and who am I to argue with it? 😆

**Comment by smira** [comment]
conventional commit should be `docs: ...`, not `fix: `, as this commit doesn't fix anything, does it?

**Comment by AlekSi** [comment]
Kind of. It "fixes" (adds back) the splash screen that makes it easier to diagnose the issue compared to the black screen. It also fixes https://github.com/talos-systems/talos/pull/3430 that was merged prematurely (can it be removed from the changelog somehow?). Not sure what prefix I should use in that situation.

File: website/content/docs/v0.10/Single Board Computers/rpi_4.md [hunk 2]
```diff
@@ -40,7 +42,8 @@ sudo bsdtar rpi-boot-eeprom-recovery-2020-09-03-vl805-000138a1.zip -C /mnt
 Remove the SD card from your local machine and insert it into the Raspberry Pi.
 Power the Raspberry Pi on, and wait at least 10 seconds.
 If successful, the green LED light will blink rapidly (forever), otherwise an error pattern will be displayed.
-If an HDMI display is attached then the screen will display green for success or red if a failure occurs.
+If an HDMI display is attached (to the port closer to the power USB-C port)
+then the screen will display green for success or red if a failure occurs.
```
Comment (rsmitty) [inline website/content/docs/v0.10/Single Board Computers/rpi_4.md:46]:
```suggestion
If an HDMI display is attached to the port closest to the power/USB-C port, the screen will display green for success or red if a failure occurs.
```

File: website/content/docs/v0.10/Single Board Computers/rpi_4.md [hunk 3]
```diff
@@ -73,6 +76,9 @@ talosctl apply-config --insecure --interactive --nodes <node IP or DNS name>
 
 Once the interactive installation is applied, the cluster will form and you can then use `kubectl`.
 
+> Note: if you have an HDMI display attached and it shows only a rainbow splash,
+> please use the other HDMI port, the one closer to the power USB-C port.
```
Comment (rsmitty) [inline website/content/docs/v0.10/Single Board Computers/rpi_4.md:80]:
```suggestion
> please use the other HDMI port, the one closest to the power USB-C port.
```

**Review by rsmitty — APPROVED** [review]

**Pull request merged**